compute/k8s-services/main.tf

```diff
@@ -150,37 +179,48 @@ module "traefik_alb_auth" {
   azure_tenant_id       = try(module.traefik_alb_auth_appreg[0].tenant_id, "")
   azure_client_id       = try(module.traefik_alb_auth_appreg[0].application_id, "")
   azure_client_secret   = try(module.traefik_alb_auth_appreg[0].application_key, "")
-  target_http_port      = var.traefik_flux_http_nodeport
-  target_admin_port     = var.traefik_flux_admin_nodeport
-  health_check_path     = "/ping"
   access_logs_bucket    = module.traefik_alb_s3_access_logs.name
+
+  # Blue variant
+  deploy            = var.traefik_alb_auth_deploy && var.traefik_flux_deploy
+  target_http_port  = var.traefik_flux_http_nodeport
+  target_admin_port = var.traefik_flux_admin_nodeport
+  health_check_path = "/ping"
+  weight            = var.traefik_flux_weight
+
+  # Green variant
+  deploy_green_variant            = var.traefik_alb_auth_deploy && var.traefik_green_variant_flux_deploy
+  green_variant_target_http_port  = var.traefik_green_variant_flux_http_nodeport
+  green_variant_target_admin_port = var.traefik_green_variant_flux_admin_nodeport
+  green_variant_health_check_path = "/ping"
+  green_variant_weight            = var.traefik_green_variant_flux_weight
 }
 
-module "traefik_alb_auth_dns" {
+module "traefik_alb_auth_dns_for_grafana" {
```

Comment:
We must also keep the DNS alias for `["internal.${var.eks_cluster_name}"]`, because it is used as a catch-all rule for anything you want to test with AUTH, and not having to create a new DNS alias for it.